src/components/post/post.pg.repository.ts

```diff
@@ -54,4 +54,9 @@ export class PgPostRepository extends PostRepository {
 
     return rows[0];
   }
+
+  @PgTransaction()
```

Comment:
Please omit this decorator. We use `@PgTransaction()` when it's needed to run multiple queries in a specific order

Reply:
```suggestion

```

Reply:
It shouldn't hurt anything to include it.

Reply:
You're right, we can leave it as it is.